### PR TITLE
feat: add min for sale price requirement

### LINKED ISSUE #253: Update the Cadastre to account for ongoing minimum `For Sale Price` & `Claim Payment` requirements
**Description**
To eliminate "dust-valued" land claims as a squatting strategy, we've implemented a minimum `For Sale Price` and `Claim Payment` for all land parcels after the Fair Launch Dutch auction is complete.  In effect, the Dutch auction ends at a configurable minimum price (>0; set in PCOLicenseParams facets) and continues in perpetuity or until the `MinForSalePrice` is changed.

**Business Requirements**
 - Update the field entry validation for _For Sale Price (ETHx)_ in the `Claim`, `Reclaim`, and `Edit` transaction windows to require a value greater than the `MinForSalePrice` and the warning to read "For Sale Price must be greater than {MinForSalePrice} ETHx"
 - Update the _For Sale Price (ETHx)_ tooltip to read as follows:
   - "Be honest about your personal valuation! A {MinForSalePrice} ETHx minimum valuation is enforced to prevent low-value squatting. ¶ You'll have 7 days to accept or reject any bid that meets this price. You must pay a penalty equal to 10% of the bidder's new For Sale Price if you wish to reject the bid. ¶ [You can read more about Partial Common Ownership in our docs.](https://docs.geoweb.network/concepts/partial-common-ownership)"
 - In the _Transaction Summary_ for `Claim` transactions:
   - `Claim Payment` should equal `MinForSalePrice`
     - Note: The user can submit a claim with a `For Sale Price` greater than the minimum, but they are only required to pay the minimum value
   - Update the hover tooltip text to read: 
"Unclaimed parcels require a minimum one-time payment of {MinForSalePrice} ETHx. This is enforced to prevent low-value claims and squatting."

**Related**
https://github.com/Geo-Web-Project/core-contracts/pull/98
https://github.com/Geo-Web-Project/core-contracts/issues/100

### PR DESCRIPTION
**Pull request by tnrdd**
# Description

Require minimum for sale price requirement for claim, edit and reclaim transactions.

# Issue

fixes #253 

# Checklist:

- [x] My commit message follows the Conventional Commits specification
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested my code
- [x] My changes generate no new warnings
- [x] My PR is rebased off the most recent `develop` branch
- [x] My PR is opened against the `develop` branch

# Alert Reviewers

@codynhat @gravenp

### CODE REVIEW

**Review by codynhat — APPROVED** [review]

File: components/cards/ActionForm.tsx [hunk 1]
```diff
@@ -340,9 +343,9 @@ export function ActionForm(props: ActionFormProps) {
               {isForSalePriceInvalid ? (
                 <Form.Control.Feedback type="invalid">
                   For Sale Price must be a number greater than{" "}
```
Comment (gravenp) [inline components/cards/ActionForm.tsx:345]: Can this text read: `For Sale Price must be greater than or equal to {" "}`

File: components/cards/ActionForm.tsx [hunk 2]
```diff
@@ -298,7 +299,9 @@ export function ActionForm(props: ActionFormProps) {
                 <InfoTooltip
                   content={
                     <div style={{ textAlign: "left" }}>
-                      Be honest about your personal valuation!
+                      Be honest about your personal valuation! A{" "}
+                      {ethers.utils.formatEther(minForSalePrice)} ETHx minimum
+                      valuation is enforced to prevent low-value squatting.
                       <br />
                       <br />
                       You'll have 7 days to accept or reject any bid that meets
```
Comment (gravenp) [inline components/cards/ActionForm.tsx:307]: Can we change the text here to `You'll have 7 days to accept or reject any bid that meets your price.`?

The change here is just from "this" to "your" price.

**Review by gravenp — CHANGES_REQUESTED** [review]
Functionality per the ticket all looks great. Thanks, @tnrdd. 

Just a couple of changes to the copy that I added in line (my bad!). 

We are handling Foreclosure Auction Claims that drop below the min price differently than how it's displaying on the Cadastre currently, but I'll create a separate ticket for that.

File: components/cards/ActionForm.tsx (hunk 1 above)
Comment (gravenp) [reply components/cards/ActionForm.tsx]: Looks good. TY

File: components/cards/ActionForm.tsx (hunk 2 above)
Comment (gravenp) [reply components/cards/ActionForm.tsx:307]: Thanks!

**Review by codynhat — APPROVED** [review]

**Pull request merged**